### ISSUE TITLE
bato.to not showing any chapters

### ISSUE THREAD

**Issue by shugenz** [TheNetsky/community-extensions#26]
everytime i try to click on any chapters, it doesn’t show up anything for me. steps to reproduce below; i’m currently using bato 3.0.3 on ios 17.2.1. much thanks!

https://github.com/TheNetsky/community-extensions/assets/158392718/a4342c5f-affe-4dd6-895d-7aa55a32a2a7

**Comment by pattypaige** [comment]
Same with mine 😭 it’s still not fixed. But the bato.to website is fine tho :((

**Comment by giaalibee** [comment]
I have the exact same problem. I was still reading bato.to fine on my phone yesterday, but now it won’t load the chapters. As for my iPad, paperback crashes whenever I try to open anything from bato.to. (Paperback v0.8.7-r12, ios 17.2.1 and on ios 15.2)

I already tried re-installing the repos and I also tried to attempt database repair but it still won’t work. Hoping this would get reaolved soon since I read most of my manhwa/ga there.
